v22.2/backup-validation.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Backup Validation
+summary: Ensure that your backups are restorable with backup validation.
+toc: true
+docs_area: manage
+---
+
+{% include_cached new-in.html version="v22.2" %} CockroachDB provides backup validation tools to ensure that backups you have in storage are restorable. You can use the following option patterns with [`SHOW BACKUP`](show-backup.html) and [`RESTORE`](restore.html) to validate a [cluster](backup.html#backup-a-cluster), [database](backup.html#backup-a-database), or [table](backup.html#backup-a-table-or-view) backup. The three options result in increasing levels of backup validation:
```

Comment:
nit: perhaps it's worth mentioning that, ultimately, the best way to validate that your backup is restorable is by actually running the restore. It could be a fourth bullet here. Or maybe the first sentence in this opening paragraph: the best way to validate a backup is restorable is by running a restore, but we provide lighter weight tools to also validate restorability.

Reply:
Thanks, good point. @msbutler I added a sentence following the opener that then leads into the options.